docs/design/fabric-v2+/fpc-cc-invocation.puml

```diff
@@ -127,14 +133,22 @@ group invoke
       end ref
 
 
-      Enclave1 -> Enclave1 : result_enc <- encrypt result with r_key
+      Enclave1 -> Enclave1 : enc_ccrsp_msg.encrypted_response <- encrypt result with r_key
       note right
         The encryption is via authenticated symmetric cipher like aes-gcm
       end note
-      Enclave1 -> Enclave1 : Sig_Enclave <- sign <args, read/writeset, result_enc> with Enclave_SK
-      return <result_enc, Sig_Enclave>
+      Enclave1 -> Enclave1 : enc_ccrsp_msg.signature <- sign <args, read/writeset, result_enc> with Enclave_SK
+      return <enc_ccrsp_msg>
```

Comment:
I am missing where `enc_ccrsp_msg.enclave_id` is set; maybe it's somewhere here but can't find it.

Reply:
well, we didn't before; added now ...